controllers/rollingupgrade_controller.go

```diff
@@ -497,16 +500,14 @@ func (r *RollingUpgradeReconciler) populateNodeList(ruObj *upgrademgrv1alpha1.Ro
 
 // Loads specific environment variables for scripts to use
 // on a given rollingUpgrade and autoscaling instance
-func loadEnvironmentVariables(ruObj *upgrademgrv1alpha1.RollingUpgrade, nodeInstance *corev1.Node) error {
+func loadEnvironmentVariables(ruObj *upgrademgrv1alpha1.RollingUpgrade, instanceID string, nodeName string) error {
 	if err := os.Setenv(asgNameKey, ruObj.Spec.AsgName); err != nil {
 		return errors.New(ruObj.Name + ": Could not load " + asgNameKey + ": " + err.Error())
 	}
-	tokens := strings.Split(nodeInstance.Spec.ProviderID, "/")
```

Comment:
Doesn't this code need to move to wherever `loadEnvironmentVariables` is called from?

Reply:
loadEnvironmentVariables() function is called only in the UpdateInstance() function. 
And in UpdateInstance() function the nodeName and InstanceID are already available. So, we won't need this line of code.